alpha/lib/model/DeliveryProfile.php

```diff
@@ -397,7 +397,7 @@ protected function getFlavorAssetInfo($url, $urlPrefix = '', $flavor = null)
 					$audioLabel = $flavor->getLabel();
 				}
 				
-				$audioLanguageData = $flavor->getLanguage() ? $this->getAudioLanguage($flavor) : null;
+				$audioLanguageData = is_callable(array($flavor, 'getLanguage')) ? $this->getAudioLanguage($flavor) : null;
```

Comment:
let do the is_callable in upper if - as currently it change the behavior in case getLanguage exist but return null